### PR TITLE
introduce get for connector settings

### LINKED ISSUE #176: Add possibility to fetch connector settings

### PR DESCRIPTION
**Pull request by pirkogdc**
closes #176

### CODE REVIEW

File: src/main/java/com/gooddata/connector/ConnectorService.java [hunk 1]
```diff
@@ -134,6 +134,35 @@ public void updateIntegration(final Project project, final ConnectorType connect
...
+    /**
+     * Get settings for given connector of given class.
+     *
+     * @param project project
+     * @param connectorType type of connector to fetch settings ofr
+     * @param settingsClass class of settings fetched
+     * @param <T> type of fetched settings
+     * @return settings of connector
+     */
+    public <T extends Settings> T getSettings(Project project, ConnectorType connectorType, Class<T> settingsClass) {
```
Comment (martiner) [inline src/main/java/com/gooddata/connector/ConnectorService.java]: can `settingsClass` be null?

**Comment by martiner** [comment]
- [ ] please create issue for this pull request

File: src/main/java/com/gooddata/connector/ConnectorService.java (hunk 1 above)
Comment (pirkogdc) [inline src/main/java/com/gooddata/connector/ConnectorService.java]: of course not

**Comment by liry** [comment]
Please add some unit tests including "unhappy path" ;)

**Comment by liry** [comment]
I'm not sure the set of these 2 public method is the right one. What about to have your getSettings private and instead of new method for every connector add public getSettings(Project project, Class<T> settingsClass) ?

**Pull request merged**